policy-format.md

```diff
@@ -0,0 +1,172 @@
+# Explainer: Format of Origin Policy documents
+
+**tl;dr** [Origin Policy](README.md) documents are JSON files with top-level
+entries for each supported policy mechanism.
+
+## General File Format
+
+An origin policy document is a JSON document that contains a series of policy
+items. A top-level dictionary names each policy items and contains item specifc
+calues. Each type of (currently) supported policy item is described below.
+
+Example:
+
+```js
+{
+  "tls": .... ,
+  "content-security-policy": .... ,
+  "referrer": ....
+}
+```
+
+## Format Versioning and Error Handling
+
+Applications should
+
+- raise an error if the file is not well-formed JSON,
+- ignore any top-level section they do not understand,
+- consider it an error if they fail to parse a policy item that they do
+  understand.
+
+**TODO:** Should there be a 'must understand' directive, so that site owners
+can determine themselves whether failure to understand a given policy item
+should be fatal ot nor?
+
+## Support Policy Items
+
+### Content Security Policy (CSP)
+
+The `content-security-policy` policy item contains the equivalent of one or more
+[Content-Security-Policy HTTP headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).
+
+- The `"content-security-policy"` policy item contains an array of
+  content-security-policy directives.
+- Each CSP directive contains a `"policy"` string, containing the CSP format.
+- A `"disposition"` is either `"enforce"` or `"report"`.
+
+Example:
+
+```js
+"content-security-policy": [
+  {
+    "policy": "frame-ancestors 'none'; object-src 'none';",
+    "disposition": "enforce"
+  },{
+    "policy": "script-src 'self' https://cdn.example.com/js/",
+    "disposition": "report",
+  }
+]
+```
+
+
+### Transport-Level Security (TLS)
+
+The `tls` policy item contains several directives related to TLS, particularly:
+
+- [HTTP Strict Transport Security](https://tools.ietf.org/html/rfc6797)
+- [HTTP Public Key Pinning](https://tools.ietf.org/html/rfc7469)
```

Comment:
Let's drop this, since Chrome is removing support and Firefox will likely follow (according to https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/he9tr7p3rZ8/eNMwKPmUBAAJ).

Reply:
Done.